classes/Language.php

```diff
@@ -1109,6 +1111,38 @@ public static function installSfLanguagePack($locale, &$errors = array())
         }
     }
 
+    /**
+     * This method has been introduce in Language out of conveniency but, if you can, prefer
+     * using the GenerateMailTemplatesService or MailTemplateGenerator services.
+     *
+     * @param array $lang_pack
+     * @param array $errors
+     */
+    public static function generateEmailsLanguagePack($lang_pack, &$errors = array())
+    {
+        $locale = $lang_pack['locale'];
+        $sfContainer = SymfonyContainer::getInstance();
+        if (null === $sfContainer) {
+            $errors[] = 'Could not generate emails because container is unavailable';
+
+            return;
+        }
+        /** @var GenerateMailTemplatesService $mailGenerator */
+        $mailGenerator = $sfContainer->get('prestashop.service.generate_mail_templates');
+        $mailTheme = Configuration::get('PS_MAIL_THEME');
+        try {
+            $mailGenerator->generateMailTemplates($mailTheme, $locale, true);
+        } catch (CoreException $e) {
+            $errors[] = Context::getContext()->getTranslator()->trans('Could not generate mail templates: %s.', array($e->getMessage()), 'Admin.International.Notification');
```

Comment:
Same here. And also, would it be possible to have this wording in the present tense?

Reply:
done

Reply:
Watch out, it must be written _Cannot blah blah_, not _Can not blah blah_. ;-)